src/lib/openapi/spec/context-field-schema.ts

```diff
@@ -5,41 +5,59 @@ export const contextFieldSchema = {
     $id: '#/components/schemas/contextFieldSchema',
     type: 'object',
     additionalProperties: false,
+    description:
+        'A [context field](https://docs.getunleash.io/reference/unleash-context) used to configure strategies with data used to evaluate isEnabled calls',
     required: ['name'],
     properties: {
         name: {
+            description: 'The name of the context field',
             type: 'string',
+            example: 'userId',
         },
         description: {
+            description: 'The description of the context field.',
             type: 'string',
             nullable: true,
+            example: 'Used to uniquely identify users',
         },
         stickiness: {
+            description:
+                'Does this context field support being used for [stickiness](https://docs.getunleash.io/reference/stickiness) calculations',
             type: 'boolean',
+            example: true,
         },
         sortOrder: {
+            description:
+                'Used to have deterministic sorting of a list of context fields. Can also be used as a draw-breaker if list of context fields is sorted alphabetically.',
             type: 'number',
+            example: 900,
         },
         createdAt: {
+            description: 'When this context field was created',
             type: 'string',
             format: 'date-time',
             nullable: true,
+            example: '2023-06-29T10:19:00.000Z',
         },
         usedInFeatures: {
             type: 'number',
```

Comment:
this and `usedInProjects` should probably also be `integers` instead of numbers: https://swagger.io/docs/specification/data-models/data-types/#numbers

Reply:
agreed, It's kinda hard to be used in 2.3 projects. Will fix.